src/utils/Cache.ts

```diff
@@ -87,4 +87,8 @@ export class Cache {
     }
     return api.storage.get(this.key);
   }
+
+  getLocale() {
```

Comment:
Naming convention that is only true for metaOverviewAbstract, so it does not work for any other cache. As the locale is in the cache key the comparision in metaOverviewAbstract.ts is not necessary. So this function is not needed in the end and can be removed again.

Reply:
Agree